packages/storage/lib/index.js

```diff
@@ -23,11 +23,18 @@ import {
   getFirebaseRoot,
 } from '@react-native-firebase/app/lib/internal';
 import StorageReference from './StorageReference';
-import StorageStatics from './StorageStatics';
+import { StringFormat, TaskEvent, TaskState } from './StorageStatics';
 import { getGsUrlParts, getHttpUrlParts, handleStorageEvent } from './utils';
 import version from './version';
 import fallBackModule from './web/RNFBStorageModule';
 
+// import { STATICS } from '@react-native-firebase/storage';
+export const statics = {
```

Comment:
I did a little fixup here to make this work (by adding the export here and removing the export below)
but the other two statics PRs would not have the `export` here, they would leave it in modular only

Reply:
good catch 👍 . I've also updated tests for static properties which I've done for others as well (using intellisense to pick up type properties as POC 😄 )

Reply:
actually, I think we should remove the static export from here, it should be from modular only in my view

Reply:
So my fixup here quieted one error but I wasn't sure on it - and I like your way of thinking better

Action - the export here should be removed, and statics should just go into the registry function for namespace attachment